Modules/_elementtree.c

```diff
@@ -1023,24 +1023,15 @@ element_setstate_from_Python(ElementObject *self, PyObject *state)
 {
     static char *kwlist[] = {PICKLED_TAG, PICKLED_ATTRIB, PICKLED_TEXT,
                              PICKLED_TAIL, PICKLED_CHILDREN, 0};
-    PyObject *args;
     PyObject *tag, *attrib, *text, *tail, *children;
-    PyObject *retval;
 
     tag = attrib = text = tail = children = NULL;
-    args = PyTuple_New(0);
-    if (!args)
-        return NULL;
-
-    if (PyArg_ParseTupleAndKeywords(args, state, "|$OOOOO", kwlist, &tag,
-                                    &attrib, &text, &tail, &children))
-        retval = element_setstate_from_attributes(self, tag, attrib, text,
-                                                  tail, children);
-    else
-        retval = NULL;
 
-    Py_DECREF(args);
-    return retval;
+    if (!PyArg_ParseTupleAndKeywords(_PyTuple_Empty, state, "|$OOOOO", kwlist,
```

Comment:
Sprinkling the code base with references to `_PyTuple_Empty` seems like a regression to me. I'd rather you keep the `PyTuple_New(0)` calls.

Reply:
Is there a good reason not to define `_PyTuple_Empty` as a public API similar to `Py_None` and `Py_True`?

Reply:
I'm in favour of using `_PyTuple_Empty` instead of `PyTupleNew(0)` internally.
It makes it explicit that it cannot fail.

No need to add it to the API unless explicitly requested by a significant body of users.